core/src/main/resources/defaults/clustertemplates/7.2.10/aws/dataengineering-ha.json

```diff
@@ -11,6 +11,7 @@
     "externalDatabase": {
       "availabilityType": "HA"
     },
+    "enableLoadBalancer": true,
```

Comment:
I thin kthis feature is under entitlement

Reply:
No Richard. This is like DL HA LB. No entitlement. Unline DL HA, DE HA template is still in "PREVIEW" state.